src/scicookie/hooks/post_gen_project.py

```diff
@@ -71,7 +71,10 @@ def remove_package_file(filepath: str):
 
 
 def move_selected_doc_dir():
-    docs_target_dir = PROJECT_DIRECTORY / "docs"
+    if DOCUMENTATION_ENGINE == "mkdocs":
```

Comment:
any reason for moving mkdocs to the root of the project, instead of the docs folder?

Reply:
Because the mkdocs getting started shows the initial structure and the mkdocs.yml in the root folder and most of the examples we reviewed are the same way.
Usually MkDocs won't let you use ., it throws a warning or gives an error. There is an extension that helps to fix this `mkdocs-same-dir` but it is best to let the user configure it if they wish.

Mainly, I made the change with users in mind, they might want to use `mkdocs build` or `mkdocs serve` instead of makim docs-preview in case they are new to this and are reading tutorials.


The error when I run mkdocs serve or mkdocs build:

`ERROR    -  Config value 'docs_dir': The 'docs_dir' should not be the parent directory of the config file. Use a child directory
            instead so that the 'docs_dir' is a sibling of the config file.
Aborted with 1 Configuration Errors!`

Reply:
thanks for the explanation

so maybe this is a bit opinionated from my side but I will try to explain my points here.

> Mainly, I made the change with users in mind, they might want to use mkdocs build or mkdocs serve instead of makim docs-preview in case they are new to this and are reading tutorials.

yes an no ... the regular user will not run mkdocs normally .. for example, for every PR for this project, have you executed jupyter-book locally? so, normally we just changes the markdown in the documentation ... and it is very easy to check the markdown format just looking the source file.

so normally the user don't need to worry to much to run mkdocs or even know that the project uses mkdocs for documentation. 

and in the case users want to  check the documentation with their changes ... if there is any info about how to build the documentation and the command with make their, that will do the trick.  

I understand your point, using the same configuration from the default recommendation is normally the best approach. but it is not the case for every project, for example, for django projects the default organization is not good enough. my point in this case to move the mkdocs to docs folder is just to keep all the docs file inside just one folder, and avoid to keep more files in the root of the project.

I am sure that is very opinionated. but at least for this PR let's keep that file inside the docs folder.

And for the next time, please let's discuss that first or at least start the discussion in the PR, instead of just changing things. and this suggestion is not just for OSL projects, it is for any project, any change in the structure of the project that is not requested by the initial issue, it is really important to discuss first.

but as I mentioned, your point is valid, but we should discuss that first, so keep the mkdocs.yaml to the docs folder please.

> ERROR - Config value 'docs_dir': The 'docs_dir' should not be the parent directory of the config file. Use a child directory instead so that the 'docs_dir' is a sibling of the config file. Aborted with 1 Configuration Errors!


this happened because some wrong configuration, with you keep the previous configuration (Makefile, mkdocs.yaml, post_gen_project.py) and the mkdocs.yaml inside the docs folder, that should work fine. I tested it locally and that worked as expected.